### PR TITLE
Consent banner stories for all services

### LINKED ISSUE #700: ConsentBanner stories need translations and knobs
**Is your feature request related to a problem? Please describe.**
Following https://github.com/bbc/psammead/pull/667 the ConsentBanner is now in Storybook. We should add stories for non-latin services including translations and typography that match the requested service. Similar to how the Paragraph stories work

**Describe the solution you'd like**
- Add ConsentBanner translation config for each service [from here](https://github.com/bbc/simorgh/tree/latest/src/app/lib/config/services)
- Add Knobs to the ConsentBanner stories that allow for swapping in and out the service

**Describe alternatives you've considered**
N/A

**Testing notes**
Manual testing only

Dev insight: Regression of storybook should be performed

**Additional context**

### PR DESCRIPTION
**Pull request by sareh**
Resolves #700

**Overall change:** Updates consent banner's storybook to have placeholder text. Can use the drop-down to select a different script. 

**Code changes:**

- Updates consent banner's storybook to have placeholder text. 
Screen Shot consent banner storybook

When selecting from the drop-down, you should be able to change the text and see placeholder text for the relevant language/script. 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by sareh** [comment]
Talking with Clare and Laura, they were happy with just having placeholder text rather than having the exact translation used in Storybook, so that's what I've implemented here.

File: packages/components/psammead-consent-banner/src/index.stories.jsx [hunk 1]
```diff
@@ -35,3 +43,38 @@ const props = {
...
+      [
+        {
+          name: 'title',
+          defaultText: 'Privacy and Cookies Policy',
+        },
+        {
+          name: 'text',
+          defaultText: 'Changes to our Privacy and Cookie Policy ',
+        },
+        { name: 'acceptText', defaultText: 'OK' },
```
Comment (ryanmccombe) [inline packages/components/psammead-consent-banner/src/index.stories.jsx]: Doesn't look like the `acceptText` or `rejectText` knobs are being used for anything - the `text` knob is being used 3 times instead

**Review by ryanmccombe — APPROVED** [review]

**Review by dr3 — CHANGES_REQUESTED** [review]
https://github.com/bbc/psammead/pull/1366/files#r305270960

**Review by dr3 — APPROVED** [review]

**Comment by sareh** [comment]
Thanks, I've just made the changes. :)

**Comment by paruchurisilpa** [comment]
Done regression on storybook and looks good to me, can be merged..

**Review by dr3 — APPROVED** [review]

**Pull request merged**